projects/facebook.com/watchman/package.yml

```diff
@@ -46,7 +46,7 @@ build:
     WATCHMAN_STATE_DIR: "{{prefix}}/var/run/watchman"
     CMAKE_ARGS:
       - -DCMAKE_INSTALL_PREFIX="{{prefix}}
-      - -DCMAKE_PREFIX_PATH=$(echo $LIBRARY_PATH | sed 's/\/lib:/:/g; s/\/lib$//')
+      - -DCMAKE_PREFIX_PATH=$(echo $LIBRARY_PATH | sed 's/\/lib:/;/g; s/\/lib$//')
```

Comment:
CMAKE_PREFIX_PATH should be fixed in pkgx 1.2.1 again. sorry for the breakage!

Reply:
Ah okay haha I was wondering what was happening...